### DISCUSSION TITLE
High tempreture model training

### DISCUSSION THREAD

**Discussion by ald3hyde** [deepmodeling/deepmd-kit#2638] [Ideas]
i want to make a model with 2000K and 300K, which has 5 elements. Now, I have a trouble that at high temperatures, certain elements are often too close to each other.
Currently, we are trying to increase the proportion of elements with short minimum distances in order to learn enough proximity distances.
Also, because of the bias in composition, there are about 70 elements A, which is enough to learn, but only about 10 elements B, which is not much to learn. So I would like to change the weights for each element (e.g., set element B to 7 times the weight of element A, etc.). Is such a thing possible with DeepMD-kit, I referred to the past Discussion and Document about atom_pref.npy, can I handle this case?

**Comment by wanghan-iapcm** [comment]
1. For multi-component system, the dpa-1 model is recommended. https://docs.deepmodeling.com/projects/deepmd/en/master/model/train-se-atten.html 
2. use [dpgen](https://github.com/deepmodeling/dpgen) to properly sample the high temperature ensemble  to avoid the close-up atoms.

**Reply by ald3hyde** [reply]
Thanks for the good suggestion. I am currently training my model using the se_e2_a descriptor. Would this be difficult?
The attention model seems to take almost a week for 1 million steps on the GPU I am using, so I would like to use a simple model if possible.
Also, you recommend me dpgen, but my DFT data is generated by a specific DFT program, not SIESTA, VASP and etc.
Can I use dpgen in such a condition?
Finally, I would like to create a model that can continually MD300k-3000k. Would ZBL be useful in this case?
If so, I would like an explanation of an example ZBL table if it is useful. I have looked at the manual and I see that set_usrtab is to specify the path to the ZBL file?

**Reply by wanghan-iapcm** [reply]
1. use se_atten and set nlayer to 0. This gives you a model with type embedding that is beneficial to multicomponent systems without the attention layer (slow). 
2. you are welcome to contribute the support of your code to dpgen
3. ZBL is usually necessary when you do radiation damage studies. The example: https://github.com/deepmodeling/deepmd-kit/pull/2613

**Reply by ald3hyde** [reply]
Thank you. I try to use se_atten.
